HealthcareSystem/PatientWebApp/Controllers/ExaminationController.cs

```diff
@@ -65,5 +65,76 @@ public ActionResult AdvanceSearchExaminations(ExaminationSearchDTO examinationSe
 
         }
 
+        /// <summary>
+        /// /getting canceled examinations linked to a certain patient
+        /// </summary>
+        /// <param name="patientJmbg">patients jmbg</param>
+        /// <returns>if alright returns code 200(Ok), if patuentJmbg is null returns 400, if connection lost returns 500</returns>
+        [HttpGet("{canceled/patientJmbg}")]
+        public ActionResult GetCanceledExaminationsByPatient(string patientJmbg)
+        {
+            try
+            {
+                List<ExaminationDTO> examinationDTOs = new List<ExaminationDTO>();
+                _examinationService.GetCanceledExaminationsByPatient(patientJmbg).ForEach(examination => examinationDTOs.Add(ExaminationMapper.ExaminationToExaminationDTO(examination)));
+                return Ok(examinationDTOs);
+            }
+            catch (NullReferenceException)
+            {
+                return BadRequest("Patient's jmbg cannot be null.");
+            }
+            catch (DatabaseException)
+            {
+                return StatusCode(500);
+            }
+        }
+
+        /// <summary>
+        /// /getting previous examinations linked to a certain patient
+        /// </summary>
+        /// <param name="patientJmbg">patients jmbg</param>
+        /// <returns>if alright returns code 200(Ok), if patuentJmbg is null returns 400, if connection lost returns 500</returns>
+        [HttpGet("{previous/patientJmbg}")]
+        public ActionResult GetPreviousExaminationsByPatient(string patientJmbg)
+        {
+            try
+            {
+                List<ExaminationDTO> examinationDTOs = new List<ExaminationDTO>();
+                _examinationService.GetPreviousExaminationsByPatient(patientJmbg).ForEach(examination => examinationDTOs.Add(ExaminationMapper.ExaminationToExaminationDTO(examination)));
+                return Ok(examinationDTOs);
+            }
+            catch (NullReferenceException)
+            {
+                return BadRequest("Patient's jmbg cannot be null.");
+            }
+            catch (DatabaseException)
+            {
+                return StatusCode(500);
+            }
+        }
+
+        /// <summary>
+        /// /getting following examinations linked to a certain patient
+        /// </summary>
+        /// <param name="patientJmbg">patients jmbg</param>
+        /// <returns>if alright returns code 200(Ok), if patuentJmbg is null returns 400, if connection lost returns 500</returns>
+        [HttpGet("{previous/patientJmbg}")]
+        public ActionResult GetFollowingExaminationsByPatient(string patientJmbg)
+        {
```

Comment:
You accidentally left the same path here as in the method for previous examinations, I assume you intended something like "following/{patientJmbg}".

Reply:
Fixed in af1b49d1baa111f3b0ee80cf8348c9c52a4adcae.